### PR TITLE
[Implement] Automatic tagging if there are any commits in the past 24 hours.

### PR DESCRIPTION
**Pull request by Rudra-IITM**
- I have introduced the "create-tag.yml" workflow, scheduled to execute daily at 12:00 am. This workflow examines the presence of commits within the last 24 hours. If such commits exist, it generates a corresponding tag with the respective date; however, in the absence of recent commits, the process refrains from creating a tag.

### CODE REVIEW

File: .github/workflows/create-tag.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Create Tag on Each Date if Any Commit
+
+on:
+  schedule:
+    # Run daily
+    - cron: '0 0 * * *'
```
Comment (tillkamppeter) [inline .github/workflows/create-tag.yml:6]: Could you choose an hour which is somewhat more "unusual"? I can imaging that many cron jobs on servers are already running at midnight. As Snap update automation is running at 7:09am in most cases, I suggest to run this job at 6:29am, shortly before Snap automation kicks in and also not on such a "usual" time.

**Comment by tillkamppeter** [comment]
Thanks for your Pull Request, please only fix the hour when it happens, all the rest is perfect.

**Review by Rudra-IITM — COMMENTED** [review]
Adjusted the scheduled run time to 6:29 AM to optimize execution during periods of reduced job traffic.

**Comment by tillkamppeter** [comment]
Merged, thanks a lot for the PR!

**Pull request merged**